src/couch_views/src/couch_views_indexer.erl

```diff
@@ -483,17 +489,28 @@ report_progress(State, UpdateType) ->
         tx_db := TxDb,
         job := Job1,
         job_data := JobData,
-        last_seq := LastSeq
+        last_seq := LastSeq,
+        db_seq := DBSeq,
+        changes_done := ChangesDone0
     } = State,
 
     #{
         <<"db_name">> := DbName,
         <<"db_uuid">> := DbUUID,
         <<"ddoc_id">> := DDocId,
         <<"sig">> := Sig,
-        <<"retries">> := Retries
+        <<"retries">> := Retries,
+        <<"active_tasks_info">> := ActiveTasks
     } = JobData,
 
+    ChangesDone1 = case maps:get(<<"changes_done">>, ActiveTasks, 0) of
```

Comment:
This can be `ChangesDone = maps:get(<<"changes_done">>, ActiveTasks, 0)` The `0` should be the value if it doesn't exist.